### PR TITLE
Update libg

### PR DESCRIPTION
**Pull request by aparajit-pratap**
### Purpose

Includes:
Fix for PolySurface-Plane intersection 

### Declarations

Check these if you believe they are true

- [x] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

### CODE REVIEW

File: src/DynamoCoreWpf/DynamoCoreWpf.csproj [hunk 1]
```diff
@@ -503,6 +503,7 @@
     <None Include="App.config">
       <SubType>Designer</SubType>
     </None>
+    <None Include="packages.config" />
```
Comment (mjkkirschner) [inline src/DynamoCoreWpf/DynamoCoreWpf.csproj:506]: 👍

Comment (mjkkirschner) [reply src/DynamoCoreWpf/DynamoCoreWpf.csproj:506]: but where is the file? I don't see it in this PR - maybe it's in the git ignore ?

Comment (aparajit-pratap) [reply src/DynamoCoreWpf/DynamoCoreWpf.csproj:506]: Yes, this file has always been ignored for some reason.

Comment (mjkkirschner) [reply src/DynamoCoreWpf/DynamoCoreWpf.csproj:506]: sorry, was your intention to add it now or should this line be removed?

**Comment by mjkkirschner** [comment]
It looks good to me, I guess it would be best to let the tests complete first though.

**Review by mjkkirschner — APPROVED** [review]

File: src/DynamoCoreWpf/DynamoCoreWpf.csproj (hunk 1 above)
Comment (aparajit-pratap) [reply src/DynamoCoreWpf/DynamoCoreWpf.csproj:506]: I'm not sure about why my packages.config file doesn't for this project doesn't list the other nuget packages. Can a project having nuget references do without a packages.config file?

Comment (mjkkirschner) [reply src/DynamoCoreWpf/DynamoCoreWpf.csproj:506]: the new style projects do use packages.config - but the old ones should have them.
I'm assuming that all the nuget references are already referenced in other project's package.config files so they're already present and it doesn't cause an issue.

**Pull request merged**